packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -200,19 +254,18 @@ export const RestrictedApplication = props => (
                           return (
                             <FetchProject projectKey={projectKeyFromUrl}>
                               {({ isLoading: isProjectLoading, project }) => {
-                                if (isProjectLoading) return null;
+                                if (isProjectLoading || !project) return null;
 
                                 // when used outside of a project context,
                                 // or when the project is expired or supsended
-                                const useProjectContext =
-                                  project &&
-                                  !(
-                                    (project.suspension &&
-                                      project.suspension.isActive) ||
-                                    (project.expiry && project.expiry.isActive)
-                                  );
+                                const useProjectContext = !(
```

Comment:
While we're here to not give a hooky impression we might want to rename this `shouldUseProjectContext`.